scripts/exrap_dump.sh

```diff
@@ -585,7 +586,7 @@ else
 
 fi
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_3} 1 proflr \
+$USHbufr_dump/bufr_dump_obs.sh $dumptime ${def_time_window_3} 1 proflr \
```

Comment:
This change is not required. The original $uscript_dump is defined in JOBSPROC_RAP_DUMP and can be retained. Please revert back to original code:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_1} 1 proflr \

Reply:
revised:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_3} 1 proflr \